sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -209,10 +221,26 @@ object ClusterBySpec {
       normalizeClusterBySpec(schema, clusterBySpec, resolver).toJson
   }
 
+  /**
+   * Converts a ClusterBySpec to a map of table properties used to store the clustering
```

Comment:
I'm confused, why do we prefer a map with only one entry over a single tuple2 like `toProperty` does?

Reply:
No preference here, this is just a bit more friendly for the call site.

Reply:
then let's be consistent here and return tuple2. The name can be `toPropertyWithoutValidation`

Reply:
Updated!